kong.yml

```diff
@@ -90,7 +91,7 @@ nginx: |
     real_ip_recursive on;
 
     # Other Settings
-    client_max_body_size 128m;
+    client_max_body_size 0m;
```

Comment:
Overriding what? We want a global cap of X, defined in the `nginx.conf`, that will always be applied, for all requests.

Behind that, @shashiranjan84 simply handles manually for API A or B the actual limit. It's not overridden.

In the documentation of the plugin we simply explain that if you set a limit above the global limit set in nginx.conf, it won't work, you need to increase the global limit in nginx.conf.